std/traits.d

```diff
@@ -2560,6 +2562,15 @@ template FieldNameTuple(T)
     static struct StaticStruct2 { int a, b; }
     static assert(FieldNameTuple!StaticStruct2 == AliasSeq!("a", "b"));
 
+    static class StaticClass1 { }
+    static assert(is(FieldNameTuple!StaticClass1 == AliasSeq!()));
```

Comment:
This test already exists four lines above?

Reply:
`class` vs. `struct`, so it's a new test.